Sources/Charcoal/CharcoalViewController.swift

```diff
@@ -333,11 +333,6 @@ extension CharcoalViewController: CalloutOverlayDelegate {
 // MARK: - UserDefaults
 
 private extension UserDefaults {
-    var regionReformCalloutShown: Bool {
-        get { return bool(forKey: "Charcoal." + #function) }
```

Comment:
We should probably be careful with this, because by changing the key in user defaults we'll show the callout for those who have already seen it.

Reply:
Yes, that definitely needs to be double checked for all things saved in user defaults before making a change like this